django/db/migrations/state.py

```diff
@@ -670,6 +671,14 @@ def __init__(self, app_label, name, fields, options=None, bases=None, managers=N
                     'ModelState.fields cannot refer to a model class - "%s.through" does. '
                     'Use a string reference instead.' % name
                 )
+            # Warn if a callable default is provided for fields having unique=True.
+            if field.unique and field.default is not NOT_PROVIDED and callable(field.default):
+                warnings.warn(
+                    'Callable defaults will not generate unique values upon migrating. '
+                    f"Found callable default {field.default} on unique field '{self.name}.{name}'. "
+                    'See https://docs.djangoproject.com/en/dev/howto/writing-migrations/'
```

Comment:
I supposed we'd want to link to either `stable` or the current `django.VERSION` instead of `dev`.

Reply:
Ah! We have `get_docs_version()`.